scripts/simplify.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/python3
```

Comment:
nit: Consider `#!/usr/bin/env python3` for better support (where python3 might be in a different path)

Reply:
Fixed 2f235019a2561ef8effb020e1e819deca2deb3f2